projects/schematics/src/migrations/5_0/methods-and-properties-deprecations/data/generated-methods-and-properties.migration.ts

```diff
@@ -1209,13 +1202,6 @@ export const GENERATED_METHODS_AND_PROPERTIES_MIGRATION: MethodPropertyDeprecati
       comment:
         "The 'getParams' method's signature changed to: 'getParams(): Observable<{\n        [_: string]: any;\n    } | undefined>'",
     },
-    {
-      class: 'DeleteUserAddressEvent',
-      importPath: '@spartacus/core',
-      deprecatedNode: 'userId',
-      comment:
-        "// TODO:Spartacus - Property 'userId' was removed from Class 'DeleteUserAddressEvent'. ",
```

Comment:
It is not removed. The `userId` is moved to parent event `UserAddressEvent`. So, the children events still have this property.